doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -500,6 +500,7 @@ Wi-Fi samples
   * :ref:`wifi_shutdown_sample` sample that demonstrates how to configure the Wi-Fi driver to shut down the Wi-Fi hardware when the Wi-Fi interface is not in use.
   * :ref:`wifi_twt_sample` sample that demonstrates how to establish TWT flow and transfer data conserving power.
   * Support for the Wi-Fi driver to several upstream Zephyr networking samples.
+  * :ref:`wifi_radio_test` sample that demonstrates how to manage re-trim parameters.
```

Comment:
replace "sample that demonstrates how to manage re-trim parameters" with "sample: Enhanced to support device re-triming process"